src/main/java/com/dumbdogdiner/stickyapi/common/webapis/MojangUser.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.webapis;
+
+import com.dumbdogdiner.stickyapi.common.user.StickyUser;
+
+import java.time.Instant;
+import java.util.*;
+
+public class MojangUser extends StickyUser {
```

Comment:
Instead of using StickyUser, just make this it's own object with the basic user information we'd need, such as username, uuid, skin textures, etc. Pretty much an object for the response from Mojang's API.

Reply:
bump